symbol-overlay.el

```diff
@@ -156,6 +156,12 @@
   :group 'symbol-overlay
   :type 'float)
 
+(defcustom symbol-overlay-cursor-hook
+  '(lambda (window oldpos entered-or-left) '())
+  "A function that is put in the cursor-sensor-functions on every overlay."
```

Comment:
When referring to an elisp symbol in the docstring, you should quote it with a backtick and an apostrophe.